script/Ciel/Day 1/01_01_CIEL01_1_3.txt

```diff
@@ -132,7 +133,7 @@ It's a palm-sized lump of metal that sparkles lustrously.
 [sha:9a946c1bff37c3f73d36e427c09ca7cbfac91b52]{
 -- Page 20, Offset 548.
 -- どう見ても携帯端末だった。
-Pretty clearly just a smartphone.
+Pretty clear it's just a smartphone.
```

Comment:
I like how the previous link with way better, though this could be improved regardless.